Add details on JSON (#941)
Codecademy/docs: content/javascript/concepts/json/json.md

  - 'paths/full-stack-engineer-career-path'
---

> In JavaScript, the reserved `JSON` object is used to convert values to JavaScript Object Notation ([JSON](https://www.codecademy.com/resources/docs/general/json)). It can also be used for parsing JSON values. `JSON` is a standard object built directly into JavaScript, so there is no need to import any external packages. `JSON` can not be constructed, and there are no functions which can create `JSON` objects.

Reviewer: ```suggestion
In JavaScript, the built-in **`JSON`** object is used to convert values to [JavaScript Object Notation (JSON)](https://www.codecademy.com/resources/docs/general/json) as well as parse those values. `JSON` is a standard JavaScript object, so there is no need to import any external packages.
```

I think the last sentence make more sense in the "Syntax" section.

## Syntax

> 

Reviewer: Could we prepend the sentence on line # 23 with the following?

"No constructor function is needed to create `JSON` objects."
